integration_tests/dbt_project.yml

```diff
@@ -1,57 +1,33 @@
 name: 'google_ads_integration_tests'
-version: '0.7.0'
+version: '0.8.0'
 profile: 'integration_tests'
 config-version: 2
 
 vars:
-  google_ads__final_url_performance: "{{ ref('google_ads_final_url_performance_data') }}"
-  google_ads__click_performance: "{{ ref('google_ads_click_performance_data') }}"
-  google_ads__criteria_performance: "{{ ref('google_ads_criteria_performance_data') }}"
-  google_ads__ad_stats: "{{ ref('google_ads_ad_stats_data') }}"
-  google_ads__ad_history: "{{ ref('google_ads_ad_history_data') }}"
-  google_ads__ad_group_history: "{{ ref('google_ads_ad_group_history_data') }}"
-  google_ads__campaign_history: "{{ ref('google_ads_campaign_history_data') }}"
-  google_ads__account_history: "{{ ref('google_ads_account_history_data') }}"
+  google_ads_schema: google_ads_integration_tests
+  google_ads_ad_stats_identifier: "ad_stats_data"
```

Comment:
Should the standard be to declare these identifiers globally or under `google_ads_source`?

Reply:
I feel like I have encountered situations where there are CircleCi issues and defining them globally has helped to reduce these errors. I think to avoid any possible errors it would be best to define globally.